CppCustomVisualizer2/CppCustomVisualizer.ChildProcessDbgSettings

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Also move to `build/targets`

Reply:
This file can't be moved. It needs to next to the .sln.

Reply:
There's no reference to this file. Will the child process debug extension be able to find this file if it's moved there?

Reply:
No, you have it in the right spot.